src/coffee/bootstrap-tour.coffee

```diff
@@ -459,7 +460,12 @@
       $template.clone().wrap('<div>').parent().html()
 
     _reflexEvent: (reflex) ->
-      if ({}).toString.call(reflex) is '[object Boolean]' then 'click' else reflex
+      if ({}).toString.call(reflex) is '[object Boolean]' or '[object String]' then 'click' else reflex
```

Comment:
`is` has higher precedence than `or`, so you need to group like `if ({}).toString.call(reflex) is ('[object Boolean]' or '[object String])'` otherwise it'll be interpreted as `if (({}).toString.call(reflex) is '[object Boolean]') or '[object String]'`.